.github/workflows/dockerhub-latest-image.yml

```diff
@@ -49,8 +46,4 @@ jobs:
       - name: build and publish images
         env:
           VERSION: latest
-          GOARCH: ${{ matrix.arch }}
-        # When we build binary for multiple platform, `go build` may be skipped as `$(SOURCES)` unchanged.
-        # So we clean the binary at first, and force to run `go build`.
-        # See https://github.com/karmada-io/karmada/pull/1649#issuecomment-1107959879 for more info.
-        run: make clean mp-image-${{ matrix.target }}
+        run: make mp-image-${{ matrix.target }}
```

Comment:
line 33 - line 38 is no need anymore? I guess we don't need to  install go, QEMU.

Reply:
and if we build images with buildx, it may cost a lot of time, so i'm confused whether do we need publist lastest image on push to master branch.

Reply:
and should emphasize the `VERSION` to `latest`?

Reply:
can also just run `make multi-platform-images`, not need to use {{ matrix.target }} to specify the component names.